.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,17 +1,15 @@
 ---
-name: Bug Report
+name: Bug Report (.NET Framework)
 about: Create a report to help us improve
 labels: bug
 ---
+-----------------Bug Template--------------------------------------------------------------
 
-**Describe your environment.** Describe any aspect of your environment relevant to the problem:
 * SDK version: 
-* .NET runtime version (.NET or .NET Core, TargetFramework in the .csproj file): 
+* .NET runtime version (.NET TargetFramework in the .csproj file):
```

Comment:
Not sure what is the best way to put this, but we need to know the application type *and* the target framework as well. If we are removing .NET or .NET Core from here, we need to put it some place else.
Specifically: If a user is having an Asp.Net Core application which targets the .NET Framework, that should be distinguished from an Asp.Net application which targets .NET Framework.